graphs/energy/nodes/industry/industry_chp_engine_gas_power_fuelmix.ad

```diff
@@ -3,6 +3,7 @@
 - use = energetic
 - presentation_group = chps
 - graph_methods = [demand, output, full_load_hours]
+- waste_outputs = [steam_hot_water]
```

Comment:
In the description of https://github.com/quintel/etsource/commit/139c40182662ea8cd545ab27c4bf8f2629fffa21 it is stated for which nodes the `steam_hot_water` attribute should be added. I think we should check whether it makes sense to add them to all CHPs now.

Reply:
See [docs](https://docs.energytransitionmodel.com/main/merit-order#marginal-costs-of-chps) for the reasoning here..